Dockerfile.docs

```diff
@@ -0,0 +1,25 @@
+# Using Python 2.7 here because mkdocs>=1.0.1 is broken on Python 3.3, 3.4, 3.7 and 3.8 :/
```

Comment:
What about 3.6? If it works on 3.6, please use that instead of 2.7.

Reply:
Sorry I tried this too! I've amended the comment to reflect this :) Only 2.7 works :/ I would love to help the upstream project fix the bugs I found but not now; maybe later :)

Reply:
Is this reflected anywhere on their docs? All I saw on their site is adding support for 3.7 in 1.0. I don’t think it’s worth contributing to mkdocs at this time. We have been discussing moving away from it for the past several months. So 2.7 is ok for now, but not a good long term decision, as Python 2.x is EOL. Let’s sync with @joelhans to determine a good long-term course of action here.

Reply:
I agree! FWIW I _think_ `mkdocs`' docs (_hmmm_) is a bit misleading somewhat and/or not as well maintained as I'd like :) It was quite painful to get this working on anything but 2.7 which is a shame. If we stick with static site generators (_which I think we should_) I can highly recommend [Hugo](https://gohugo.io/) here; but I do not know how well it will fit specifically in our use case for how we've llaid out our docs and presentation.